components/header/header.js

```diff
@@ -0,0 +1,72 @@
+import "../link-button/link-button.js";
+
+const template = document.createElement("template");
+template.innerHTML = `
+  <style>
+    :host {
+      position: fixed;
+      top: 0;
+      left: 0;
+      padding: 20px 0;
+      width: 100%;
+      border-bottom: 1px solid #D3D3D3;
+      background-color: rgba(255, 255, 255, 0.8);
+    }
+
+    header {
+      width: 80%;
+      margin: 0 auto;
+      justify-content: space-between;
+      display: flex;
+      align-items: center;
+    }
+
+    .buttons-container {
+      display: flex;
+    }
+
+    .logo {
+      display: flex;
+      gap: 10px;
+      align-items: center;
+      text-decoration: none;
+    }
+
+    .logo img {
+      width: 45px;
+      height: 22.5px;
+    }
+
+    .logo span {
+      font-size: 20px;
+      font-weight: 500;
+      color: black;
+    }
+
+    .logo img,
+    .logo span {
+    vertical-align: middle;
+    }
```

Comment:
My bad. Thanks for pointing this out. I did a manual formatting since the prettier didnt affect when css is in js file. This is one of the disadvantages of putting all css into the js file i think. Or Ill look for a workaround to utilize the prettier in this case.